usr/bin/fang-rtsp-watchdog.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+echo "RTSP Watchdog script started"
+
+while :
+do
+        sleep 10
+        if pgrep -x "snx_rtsp_server" > /dev/null
```

Comment:
This will have the 'problem' that it will start the RTSP server even if you manually stopped it.

Sorry for resurfacing this, but I implemented the exact same script before checking the open pull requests, and I had that issue.

Maybe you could check the existence of the `/var/run/rtsp-server.pid` file and check the enclosed PID?